documentation/man/kw.rst

```diff
@@ -0,0 +1,217 @@
+
+2. Remove the requirement for password in the VM to became root.
+
+COMMANDS FOR WORKING WITH CODE
+==============================
+
+Projects that have a similar workflow of Linux Kernel, usually have a set of
+tools that simplify part of the tasks related with the code. This section
+describes some of the key features supported by **kw** for helping with code.
+
+b, build
+--------
+This command expects a local Makefile to build the project; it takes cares of
+the correct number of jobs to be used for your machine. This command tries to
+take advantage of your hardware, and try to use all of your CPU cores.
+
+c, codestyle [*DIRECTORY|FILE*]
+-------------------------------------------
+The **checkpatch** tool is one of the most fabulous scripts provided by the
+Linux Kernel to help developers to follow the code style adopted by the
+project. This command is a wrapper for **checkpatch** that aims to simplify the
+use of this tool; notice that you can specify a single file or an entire
+directory.
+
+e, explore [log] [*EXPRESSION*] [-p] [*DIRECTORY|FILE*]
+-------------------------------------------------------
+The *explore* command can search for string match in the repository and the git
+log messages. For example, you can use **kw e functionName** to find
+functionName in the source directory. You can also search the log history by
+using *log* after the *e*; for instance, **kw e log STRING_MATCH**.
+Additionally, you can use *-p* to see the diff in the search.
+
+OTHER COMMANDS
+==============
+
+Manage configuration files it is part of many projects; this section describes
+a tool available in the **kw** to help developers to keep track of such config
+files. Additionally, there is many other features that **kw** provides but that
+not fit in the previous section; here you can find all the other elements.
+
+g, configm [*--save NAME* [*-d DESCRIPTION*][-f]]|[*--ls*]
+----------------------------------------------------------
+
+The 'configm' option it is the application that manages the '.config' files for
+users. In summary, it provides operations for save, load, removes, and list
+'.config' previously saved by the user. See the current options:
+
+1.  --save NAME [-d DESCRIPTION] [-f]: The save option seeks in the current
+directory for a '.config' file to be added under the management of kw.  The
+save option expects a name to be used as an alias for the target config file.
+If we have a local .config and a valid name, kw saves the configuration file.
+Additionally, users can add a description by using *-d* flag. Finally, if the
+user tries to add the same name twice kw will warn about it; the '-f' will
+suppress this message.
+
+2. --ls list all the config files available.
+
+v, vars
+-------
+Show variable status used by **kw**.
+
+bi
+--
+Build and install a new module version. It is a combination of **build** and
+**install**.
+
+h, help
+-------
+Show basic help.
+
+ABOUT kworflow.config
```

Comment:
Although it is implicit, I think we should explicitly say at the text that the user should add local config files for his/her projects as desired, but he can also change the global config file to have a custom kw configuration running for every directory he/she whishes to invoke the application. Also, we should mention where the global file is.